### PR TITLE
Fix Already define exception when using --var="foo=bar"

### PR DESCRIPTION
**Pull request by cedeyn**
This PR fixes :
$ milkcheck -c ~/devel/milkcheck-cedeyn/conf/test_variables status --var 'foo=bar'
[14:36:42] ERROR    -

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,5 @@
 language: python
 python:
-    - "2.6"
```
Comment (degremont) [inline .travis.yml]: If supports for 2.6 is really dropped, this worths a separated PR with doc updates

File: lib/MilkCheck/ServiceManager.py [hunk 2]
```diff
@@ -76,7 +76,10 @@ def _variable_config(self, conf):
             for defines in conf.get('defines', []):
                 for define in defines.split():
                     key, value = define.split('=', 1)
-                    self.add_var(key, value)
+                    try:
+                        self.add_var(key, value)
+                    except VariableAlreadyExistError:
```
Comment (degremont) [inline lib/MilkCheck/ServiceManager.py]: i'm surprised we never have this use case before...?

File: lib/MilkCheck/ServiceManager.py [hunk 3]
```diff
@@ -148,14 +151,15 @@ def call_services(self, services, action, conf=None):
...
-        # Create global variable from configuration
-        self._variable_config(conf)
         if conf:
             # Apply configuration over the graph
             self._apply_config(conf)
             # Enable reverse mode if needed, based on config
             self.algo_reversed = action in conf.get('reverse_actions')
 
+        # Create global variable from configuration
+        self._variable_config(conf)
```
Comment (degremont) [inline lib/MilkCheck/ServiceManager.py]: I'm remember this position in the code being special. I'm also surprised everything is fine.
Are you sure there are no side effects?

By example, it looks like excluded_nodes will never be set now that this method is called later in the code

Comment (cedeyn) [reply lib/MilkCheck/ServiceManager.py]: Tests are fine and i tested it manually and nodes are correctly excluded. Same for services.

Comment (degremont) [reply lib/MilkCheck/ServiceManager.py]: Ok, let's say it was some time ago when the code was different and now it is fine... :/
You will see soon :)

File: .travis.yml (hunk 1 above)
Comment (degremont) [reply .travis.yml]: and what about this one?

Comment (cedeyn) [reply .travis.yml]: It's the travis support of python 2.6 which is dropped, not the milkcheck one.
Without this commit, travis complains.

Comment (degremont) [reply .travis.yml]: aaaah ok

**Pull request merged**